packages/components/src/Loc.js

```diff
@@ -0,0 +1,23 @@
+import { makeLocalizedElement } from './make-localized-element';
+import { STANDARD_ELEMENT_TYPES } from './constants';
+
+export const Loc = {
+  A: makeLocalizedElement('a'),
```

Comment:
What's the reason for capitalizing these? Why not match the actual element name exactly?

Reply:
Interesting idea!

My reasoning for the capitalization was to make clear that it is a React component, and not just the basic html element. I think if we figure out a clear way to resolve this issue https://github.com/udacity/fluent-react-utils/issues/9, then having the capitals will be particularly beneficial to understand that it's different. If we don't, I could see the argument for lowercase, since it would just be `<Loc.a>`. Though l still feel like it goes against [React best practices](https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized)